pkg/clients/generated/apis/bigqueryconnection/v1alpha1/bigqueryconnectionconnection_types.go

```diff
@@ -185,6 +122,18 @@ type BigQueryConnectionConnectionSpec struct {
 	ResourceID *string `json:"resourceID,omitempty"`
 }
 
+type ConnectionCloudResourceStatus struct {
+	/* The account ID of the service created for the purpose of this connection. */
+	// +optional
+	ServiceAccountId *string `json:"serviceAccountId,omitempty"`
```

Comment:
Is this a Google service account? If so, can you name it `gcpServiceAccount` to distinguish with the Kubernetes service account (which in CRD is normally called `ServiceAccount`)?

Reply:
Same as above